changehc/delphi_changehc/run.py

```diff
@@ -180,12 +181,21 @@ def run_module(params: Dict[str, Dict[str, Any]]):
                 elif numtype == "cli":
                     data = load_cli_data(file_dict["denom"],file_dict["flu"],file_dict["mixed"],
                              file_dict["flu_like"],file_dict["covid_like"],dropdate_dt,"fips")
-                su_inst.update_sensor(
+                more_stats = su_inst.update_sensor(
                     data,
                     params["common"]["export_dir"]
                 )
+                stats.extend(more_stats)
```

Comment:
`update_sensor` returns the output of `create_export_csv`, which is a pd.Series of dates. Is there some tuple conversion somewhere that I missed?